.github/workflows/dependency-update.go-mod-tidy_and_make-package-specs.yaml

```diff
@@ -1,13 +1,14 @@
 name: "renovate-/ospo-bot: go mod tidy and make package-specs"
 on:
-  push:
+  [pull_request, push]:
```

Comment:
I don't think a _sequence_ can be a _mapping key_: https://yaml.org/spec/1.2.2/#rule-c-mapping-key

Reply:
I think you are right. Unless I want to use referencing in yaml (which does not always work with every tool), we need to duplicate our configuration.